configuration/configuration.go

```diff
@@ -24,42 +25,43 @@ type ApplicationCluster struct {
 				Execute struct {
 					Shell   string `yaml:"Shell"`
 					Kubectl struct {
-						Path    string `yaml:"Path"`
-						Type    string `yaml:"Type"`
+						Path string `yaml:"Path"`
+						Type string `yaml:"Type"`
 					} `yaml:"Kubectl"`
 				} `yaml:"Execute"`
 			} `yaml:"Action"`
 		} `yaml:"Application"`
 	} `yaml:"Applications"`
 }
 type ProviderCluster struct {
-		InitialNodeCount int    `yaml:"InitialNodeCount"`
-		InitialNodeType  string `yaml:"InitialNodeType"`
-		Name             string `yaml:"Name"`
-		Project 		 string `yaml:"Project"`
-		NodePools        []struct {
-			NodePool struct {
-				Count    int    `yaml:"Count"`
-				Name     string `yaml:"Name"`
-				NodeType string `yaml:"NodeType"`
-				Labels string `yaml:"Labels"`
-			} `yaml:"NodePool"`
-		} `yaml:"NodePools"`
-		OauthScopes     string `yaml:"OauthScopes"`
-		PostInstallHook []struct {
-			Execute struct {
-				Shell string `yaml:"Shell"`
-				Path string `yaml:"Path"`
-			} `yaml:"Execute"`
-		} `yaml:"PostInstallHook"`
-		PostDeleteHooak []struct {
+	InitialNodeCount int    `yaml:"InitialNodeCount"`
+	InitialNodeType  string `yaml:"InitialNodeType"`
+	Labels           string `yaml:"Labels"`
+	Name             string `yaml:"Name"`
+	Project          string `yaml:"Project"`
+	NodePools        []struct {
+		NodePool struct {
+			Count    int    `yaml:"Count"`
+			Name     string `yaml:"Name"`
+			NodeType string `yaml:"NodeType"`
+			Labels   string `yaml:"Labels"`
```

Comment:
Why not:
```
Labels map[string]string `yaml:"labels"`
```

Reply:
I have just done a copy of the code wrote by Alex